modules/features/pages/gpt.adoc

```diff
@@ -1,3 +1,134 @@
 = Golden Path Templates
 
-This is _Section 2_ of _Chapter 3_ in the *hello* quick course....
\ No newline at end of file
+**Golden Path Templates** (GPT), also known as **Software Templates**, are pre-defined, standardized configurations and workflows that serve as a starting point for new projects or services. They are typically created and maintained by platform engineers, and consumed by developers.
+
+GPTs typically include recommended configurations for various aspects of the application or service, such as application metadata, linters and static analysis tools, documentation, CI/CD pipelines, testing frameworks, security scanning tools and libraries, and many more features. GPTs enable true self-service for developers. They ensure consistency and adherence to organizational standards across different services or projects.
+
+It is important to understand that GPTs should not be seen as yet another process pushed down to developers from a centralized authority. They are a way to encapsulate the various different tools, processes and utilities used in the organization and compose them together into a single tool capable of quickly scaffolding new projects, products and services of any kind. They are not meant to replace your already existing automation tools, but to complement them.
+
+== Creating Golden Path Templates
+
+There are three different ways to create templates:
+
+=== Manually using YAML Files
+
+Templates are defined in a YAML file with the **kind: Template** attribute. The template defines the set of scaffolded files, along with parameters and variables that capture input to the template.
```

Comment:
Could you maybe elaborate a bit what you mean with "template defines the set of scaffolded files" files?

Templates refers to a repo that contains files (with optional placeholders) that are processed during scaffolding. Is that what you refer to in this context?

Reply:
fixed